packages/expect-type/src/__tests__/index.test.ts

```diff
@@ -69,6 +69,8 @@ test('Test for basic javascript types', () => {
   expectTypeOf('').toBeString()
   expectTypeOf(1).toBeNumber()
   expectTypeOf(true).toBeBoolean()
+  // eslint-disable-next-line no-empty-function
```

Comment:
The lint task makes sure this test gets copied to the readme - so can we make this an eslint-disable at the top of the file instead? The copying should happen automatically and fix the lint failure if you run `rush lint --fix` after that.

Reply:
Hm not sure I understood (kinda new to Rush too), the lint rule should go at the top of the .ts rather than just for the next line?

Reply:
Ah wait I think I understood, the actual test gets copied so it takes the lint rule with it and it messes with the actual linter? 😅

Reply:
Yes it's something like that! Yes I'm trying out rush in this repo so some of the workflows are still awkward. You _can_ lint without it but rush manages the lockfile so easiest to let it do its thing.

Reply:
Nice, should be good now I think, let me know if there are other changes needed.
Great lib by the way 👍